docs/operator.md

```diff
@@ -333,3 +335,29 @@ EOF
 ```
 
 For more details on how to specify the CR, see [AgentServiceConfig CRD](https://github.com/openshift/assisted-service/blob/master/config/crd/bases/agent-install.openshift.io_agentserviceconfigs.yaml).
+
+### Image Registries Without Authentication
+
+`assisted-service` validates the [pull secret](hive-integration/kube-api-getting-started.md#2-create-a-pull-secret) provided for spoke cluster installation by making sure it contains the authentication information for every image registry used for installation. 
+
+Registries that don't require authentication can be added to an ignore list (called `PUBLIC_CONTAINER_REGISTRIES` in `assisted-service`). Any registry on the ignore list is not required to have an entry in the pull secret validated by the `assisted-service`.
```

Comment:
Isn't it sort of an implementation details at this point? I mean, maybe we can rephrase this to emphasize that the new API ('unauthenticatedRegistries') should be used. So in line 363, we can say that the 'ConfigMap override' is still supported for backwards compatibility. wdyt?

Reply:
Good point! I've reworded as suggested, how does it sound?

Reply:
looks good now:)